### ISSUE TITLE
Check if server-side periodic tasks are correctly executed

### ISSUE THREAD

**Issue by mtog** [1001genomes/AraGWAS#95]
Check all periodic tasks (temp file cleaning, default file generations)

**Comment by mtog** [comment]
3ca48f5221df38de5d982ff7178db65934ed25e0 contributes to this. Manual check would be ideal. We will assess that once the final version deploying preloaded JSON file from aragwas_data is up and running.

**Comment by timeu** [comment]
the periodic tasks do run however due to some file permission issues they currently fail. I need to fix that before.

**Comment by timeu** [comment]
Fixed the file permission issue and will check tomorrow if they successfully ran

**Comment by timeu** [comment]
@mtog : https://github.com/1001genomes/AraGWAS/blob/3ca48f5221df38de5d982ff7178db65934ed25e0/aragwas_server/gwasdb/es2csv.py#L190

This will append to the existing file. This might not be the best option. We should probably use the `w+`  option which will truncate the file.

**Comment by mtog** [comment]
Sorry I confused this function with the above use of temp_file for dumping the data. Changed it.

**Comment by timeu** [comment]
yeah I changed the tmp folder location because the problem is that the tmp is different for the worker container which runs all the tasks and the backend server. The only thing that is shared it the HDF5_FILE_FOLDER. Actually a more reliable way of doing is to generate the file in a tmp folder (that might be local to the container) and then copy it to the HDF5_FILE_FOLDER. this way we can also make sure that the csv works before overwriting the old file. But for now this solution is fine

**Comment by mtog** [comment]
Perfect, thank you. I fixed a small typo in the csv file name that prevented download. 
We can keep this issue open to change to the more reliable way later on.

**Comment by mtog** [comment]
It seems that this is working, can we close this @timeu ?
